popout.js

```diff
@@ -0,0 +1,35 @@
+//Runs in the settings.html page when the user clicks the "Generate Browser Source Tracker" button. Generates a popout window and passes params.
+function initBrowserSource(){
```

Comment:
This should be a .mjs file

Reply:
Anything I need to do besides rename the file? Should it go in that js folder with the other .mjs files or anything?

Reply:
yeah in the js/ folder. mjs are a little different. they load asynchronously. and need to export functions used by other scripts. and can only be called by other js module code.

Reply:
Alright I've converted it and adjusted the settings/popout pages accordingly, thanks for explaining how it works